lib/saml/url_service.rb

```diff
@@ -138,8 +141,14 @@ def verify_url
           build_authn_context('myhealthevet_loa3')
         when 'dslogon', 'dslogon_multifactor'
           build_authn_context('dslogon_loa3')
+        when IAL::LOGIN_GOV_IAL1
+          build_authn_context([IAL::LOGIN_GOV_IAL2, AAL::LOGIN_GOV_AAL2], AuthnContext::LOGIN_GOV)
         when SAML::UserAttributes::SSOe::INBOUND_AUTHN_CONTEXT
```

Comment:
This is fine for this PR, but I'm worried about using `SAML::UserAttributes::SSOe::INBOUND_AUTHN_CONTEXT` going forward. Can we create a ticket to remove that and refactor this line?

Reply:
Right, plan on bringing it up at the meeting later today.